lib/screens/lobby.dart

```diff
@@ -3,16 +3,49 @@ import 'package:niira/services/auth/auth_service.dart';
 import 'package:provider/provider.dart';
 
 class LobbyScreen extends StatelessWidget {
-  const LobbyScreen({Key key}) : super(key: key);
-
   @override
   Widget build(BuildContext context) {
+    Future<void> _confirmSignOutDialog() async {
```

Comment:
Do you think it would be worth making a ConfirmationDialog class or something similar, something that takes a message to display and returns the user response - and a function in the NavigationService to display the ConfirmationDialog, so it can be called from anywhere?